multi-node/aws/pkg/config/templates/cloud-config-worker

```diff
@@ -13,19 +13,39 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
 
     - name: kubelet.service
       enable: true
       command: start
       content: |
+{{ if eq .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=docker.service load-rkt-stage1.service
+        After=docker.service load-rkt-stage1.service
+{{ end }}
         [Service]
         Environment=KUBELET_VERSION={{.K8sVer}}
         Environment=KUBELET_ACI={{.HyperkubeImageRepo}}
-        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf --mount volume=dns,target=/etc/resolv.conf"
+        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf \
+        --mount volume=dns,target=/etc/resolv.conf \
+        --volume=mp,kind=host,source=/usr/sbin/modprobe \
```

Comment:
I misinterpreted, you were asking about modprobe not the existing resolv.conf stuff.

https://github.com/kubernetes/kubernetes/blob/f2ddd60eb9e7e9e29f7a105a9a8fa020042e8e52/pkg/kubelet/network/plugins.go#L159

I believe I ran into this ^^ at some point.